src/test/java/org/kainos/ea/integration/JobIntegrationTest.java

```diff
@@ -0,0 +1,43 @@
+package org.kainos.ea.integration;
+
+import java.util.List;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.DropwizardTheITCrowdServiceApplication;
+import org.kainos.ea.DropwizardTheITCrowdServiceConfiguration;
+import org.kainos.ea.cli.Job;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class JobIntegrationTest {
+
+    private static final String host = System.getenv("BASE_URL");
+
+    static final DropwizardAppExtension<DropwizardTheITCrowdServiceConfiguration> APP = new DropwizardAppExtension<>(
+            DropwizardTheITCrowdServiceApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+
+    @Test
+    void getJobRoles_shouldReturnListOfJobRoles() {
+
+        // Construct the target URL with the host URL
+        String targetUrl = host + "/api/jobs";
```

Comment:
Pipeline Seems To Failing With URL Encoding. Env is set -> Still fails. May need to use?

String targetUrl = URLEncoder.encode(host + "/api/jobs", "UTF-8")